lib/Config/TranslationUpdater.php

```diff
@@ -77,10 +77,12 @@ private function getTranslationUpdates(array $locales): array {
       $requestBody['plugins'][$this->premiumSlug] = ['version' => $this->premiumVersion];
     }
 
+    // Three seconds, plus one extra second for every 10 plugins.
```

Comment:
I think you mean `for every 10 locales` don't you?

Reply:
Yes, you are right.